examples/gzip/gzip-envoy.yaml

```diff
@@ -37,6 +37,21 @@ static_resources:
                   "@type": type.googleapis.com/envoy.extensions.compression.gzip.compressor.v3.Gzip
                   memory_level: 3
                   window_bits: 10
+          - name: envoy.filters.http.decompressor
+            typed_config:
+              "@type": type.googleapis.com/envoy.extensions.filters.http.decompressor.v3.Decompressor
+              decompressor_library:
+                name: upload
+                typed_config:
+                  "@type": "type.googleapis.com/envoy.extensions.compression.gzip.decompressor.v3.Gzip"
+                  window_bits: 9
+                  chunk_size: 8192
+                  max_inflate_ratio: 1000
```

Comment:
i think we should probably say something about the decompression ratio - as we know - if this is set too low (for our expectations) then it will fail

Reply:
OK, I added a comment in yaml file.Thanks!